services/app/src/components/gameFooter/GameFooter.jsx

```diff
@@ -18,9 +18,12 @@ function GameFooter({ skipCurrentPlayerMove }) {
         <button type="button" className="finish-btn" disabled={!lastTurn || isAllTurnsMade || (finished && !isAllTurnsMade)} onClick={finishMove}>
           Finish Move
         </button>
-        <button type="button" className="skip-btn" disabled={finished} onClick={() => skipCurrentPlayerMove()}>
-          Skip Move
-        </button>
+        <div className='hoverText'>
```

Comment:
Please maintain the class naming conventions by using `kebab-case` naming for class names, e.g. `hover-text`, `expl-text`, etc.

Reply:
Apart from that, the functionality we're building here is esentially a `tooltip` which is quite common. I think we should make it reusable.
The `Tooltip` component should be created. It should take a `title` as a prop and should be used like this:
```
<Tooltip title="Skips your move.">
  <button .../>
</Tooltip>
```